tests/system/model_monitoring/test_model_monitoring.py

```diff
@@ -60,13 +60,133 @@ class TestModelEndpointsOperations(TestMLRunSystem):
+                pytest.skip(
+                    "taosws:// is not set as MLRUN_MODEL_ENDPOINT_MONITORING__TSDB_CONNECTION, skipping."
+                )
+
+        tsdb_client = mlrun.model_monitoring.get_tsdb_connector(
+            project=self.project_name,
+            tsdb_connection_string=tsdb_connection_string,
+        )
+        try:
+            self.project.set_model_monitoring_credentials(
+                stream_path=mlrun.mlconf.model_endpoint_monitoring.stream_connection,
+                tsdb_connection=tsdb_connection_string,
+            )
+            db = mlrun.get_run_db()
+            model_endpoint = self._mock_random_endpoint("testing")
+            model_endpoint = db.create_model_endpoint(model_endpoint)
+
+            model_endpoint2 = self._mock_random_endpoint("testing2")
+            model_endpoint2 = db.create_model_endpoint(model_endpoint2)
+
+            mep_uid = model_endpoint.metadata.uid
+            mep2_uid = model_endpoint2.metadata.uid
+            tsdb_client.create_tables()
+            tsdb_client.write_application_event(
+                self._generate_event(endpoint_id=mep_uid, result_name="result1")
+            )
+            tsdb_client.write_application_event(
+                self._generate_event(endpoint_id=mep_uid, result_name="result2")
+            )
+            tsdb_client.write_application_event(
+                self._generate_event(
+                    endpoint_id=mep_uid, result_name="metric1", event_kind="metric"
+                )
+            )
+            tsdb_client.write_application_event(
+                self._generate_event(endpoint_id=mep2_uid, result_name="result3")
+            )
+            expected_for_mep1 = ["invocations", "metric1", "result1", "result2"]
+            expected_for_mep2 = ["invocations", "result3"]
+
+            income_events_mep1 = self._run_db.get_model_endpoint_monitoring_metrics(
+                project=self.project.name, endpoint_id=mep_uid
+            )
+            assert expected_for_mep1 == sorted(
+                [event.name for event in income_events_mep1]
+            )
+            income_events_by_endpoint = self._run_db.get_metrics_by_multiple_endpoints(
+                project=self.project.name, endpoint_ids=[mep_uid, mep2_uid]
+            )
+
+            result_for_mep1 = [
+                event.name for event in income_events_by_endpoint[mep_uid]
+            ]
+            assert expected_for_mep1 == sorted(result_for_mep1)
+
+            result_for_mep2 = [
+                event.name for event in income_events_by_endpoint[mep2_uid]
+            ]
+            assert expected_for_mep2 == sorted(result_for_mep2)
+
+            #  asserts for non-exist mep_ids
+            result_for_non_exist = self._run_db.get_metrics_by_multiple_endpoints(
+                project=self.project.name, endpoint_ids=["not_exist"], type="results"
+            )
+            assert not result_for_non_exist["not_exist"]
```

Comment:
Better be explicit about the expected result (`None`? `[]`?). `assert not x` will cause an empty `AssertionError` that doesn't actually say what was the offending value. Applies elsewhere as well.